server.js

```diff
@@ -1,11 +1,18 @@
 //Lets require/import the HTTP module
 var http = require('http');
+var fs = require('fs');
  //Lets define a port we want to listen to
 var PORT=8080;
  //We need a function which handles requests and send response
 function handleRequest(request, response){
-    response.end('It Works!! Path Hit: ' + request.url);
```

Comment:
this line 7 `response.end('It Works!! Path Hit: ' + request.url);` should be deleted. It's replaced by the line 10 thru 14 which serves the index.html file instead of printing a string to the DOM.